mllib/src/test/scala/org/apache/spark/ml/classification/DecisionTreeClassifierSuite.scala

```diff
@@ -398,6 +398,14 @@ class DecisionTreeClassifierSuite
 
     testDefaultReadWrite(model)
   }
+
+  test("intermediate dataset storage level") {
```

Comment:
Ideally we need to test that the actual storage levels for the intermediate RDDs are correct. I did this for ALS - not sure if the same approach might be used here.